config/locales/en.yml

```diff
@@ -47,8 +47,8 @@ en:
             style_in_template: "<style> tag in %{template}. Use an app.css file instead."
             no_code_for_ifo_notemplate: Javascripts, stylesheets, and templates are
               not allowed when an iframe URI or noTemplate is specified
-            invalid_default_locale: "%{defaultLocale} is not a valid default locale."
-            missing_translation_file: 'Missing translation file for locale ''%{defaultLocale}''.
+            invalid_default_locale: "%{default_locale} is not a valid default locale."
```

Comment:
thanks for looking @mariasica I've pushed a couple more I18n changes to this branch, it's now ready to merge with your +1